app/models/imported_clients_csv.rb

```diff
@@ -61,13 +61,14 @@ def import(agency)
           client.available = false
           client.identified = true
         end
+        assessment = client.current_assessment || client.client_assessments.build
+
         timestamp = convert_to_time(row[FORM_TIMESTAMP])
-        if client.imported_timestamp.nil? || timestamp > client.imported_timestamp
+        if assessment.imported_timestamp.nil? || timestamp > assessment.imported_timestamp
           @clients << client
-          @touched += 1 if client.imported_timestamp.present?
-          client.update(
+          @touched += 1 if assessment.imported_timestamp.present?
+          assessment.update_attributes(
```

Comment:
`update` and `update_attributes` are equivalent.  Did you mean `update_columns`? https://davidverhasselt.com/set-attributes-in-activerecord/